com.unity.render-pipelines.universal/Runtime/Debug/DebugRenderSetup.cs

```diff
@@ -0,0 +1,208 @@
+
+using System;
+using UnityEditor.Rendering;
+
+namespace UnityEngine.Rendering.Universal
+{
+    public class DebugRenderSetup : IDisposable
```

Comment:
Consider if this should be public.

Reply:
Also overlooked by me, most likely can be made internal.

Reply:
Made internal